src/core/conversation.py

```diff
@@ -58,6 +58,7 @@ def _send_message_analysis(self) -> None:
         self._add_to_conversation(
             ConversationRolesInternalEnum.ANALYSIS, analysis_response
         )
+        self._runtime.add_description(analysis_response)
```

Comment:
In the next step I we should exclude 'analysis' step from the message and include only it